src/spikeinterface/qualitymetrics/quality_metric_list.py

```diff
@@ -66,7 +66,11 @@
     "amplitude_cutoff": ["amplitude_cutoff"],
     "amplitude_median": ["amplitude_median"],
     "amplitude_cv": ["amplitude_cv_median", "amplitude_cv_range"],
-    "synchrony": ["sync_spike_2", "sync_spike_4", "sync_spike_8"],
+    "synchrony": [
+        "sync_spike_2",
+        "sync_spike_4",
+        "sync_spike_8",
+    ],  # we probably shouldn't hard code this. This is determined by the arguments in the function...
```

Comment:
but I think we agreed that we can just hard-code this at the QM level, so it should be ok.

Let's keep the comment until this is actually hard-coded!

Reply:
Actually, this was done already: https://github.com/SpikeInterface/spikeinterface/pull/3559